website/content/en/preview/AWS/provisioning.md

```diff
@@ -186,6 +186,7 @@ spec:
           kmsKeyID: "arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab"
           deleteOnTermination: true
           throughput: 125
+          snapshotID: snap-0123456789

```

Comment:
Can we leave this out of the docs, since I'd expect users would typically not specify this.

Reply:
I think it's important to document all the fields supported here since we don't have docs elsewhere that are more detailed. I suppose you could look it up in the CRD but this is definitely more convenient.

Reply:
@bwagner5 On a (kinda) similar topic... I was looking in the official docs about `Trhoughput` parameter and I couldn't find anything on it. Finally in Karpenter's source code I found out this comment `Throughput to provision for a gp3 volume, with a maximum of 1,000 MiB/s`. Is this documented anywhere in official docs?